druid/ingestion-spec/ml_observation_evidence_kafka.json

```diff
@@ -0,0 +1,125 @@
+  "dataSchema": {
+    "dataSource": "sl-observation-evidence",
+    "parser": {
+      "type": "string",
+      "parseSpec": {
+        "format": "json",
+        "timestampSpec": {
+          "column": "completedDate",
+          "format": "auto"
+        },
+        "dimensionsSpec": {
+          "dimensions": [
+            {
+              "type": "string",
+              "name": "total_evidences"
+            },
+            {
+              "type": "string",
+              "name": "observationSubmissionId"
+            },
+            {
+              "type": "string",
+              "name": "entity"
+            },
+            {
+              "type": "string",
+              "name": "entityExternalId"
+            },
+            {
+              "type": "string",
+              "name": "entityName"
+            },
+            {
+              "type": "string",
+              "name": "entityTypeId"
+            },
+            {
+              "type": "string",
+              "name": "entityType"
+            },
+            {
+              "type": "string",
+              "name": "createdBy"
+            },
+            {
+              "type": "string",
+              "name": "solutionExternalId"
+            },
+            {
+              "type": "string",
+              "name": "solutionId"
+            },
+            {
+              "type": "string",
+              "name": "observationId"
+            },
+            {
+              "type": "string",
+              "name": "remarks"
+            },
+            {
+              "type": "string",
+              "name": "questionId"
+            },
+            {
+              "type": "string",
+              "name": "questionExternalId"
+            },
+            {
+              "type": "string",
+              "name": "questionResponseType"
+            },
+            {
+              "type": "string",
+              "name": "questionName"
+            },
+            {
+              "type": "string",
+              "name": "evidence_count"
+            },
+            {
+              "type": "string",
+              "name": "fileName"
+            },
+            {
+              "type": "string",
+              "name": "fileSourcePath"
+            },
+            {
+              "type": "string",
+              "name": "completedDate"
+            },
+            {
+              "type": "string",
+              "name": "appName"
+            }
+          ]
+        }
+      }
+    },
+    "metricsSpec": [],
+    "granularitySpec": {
+      "type": "uniform",
+      "segmentGranularity": "DAY",
+      "queryGranularity": "none",
+      "rollup": false
+    }
+  },
+  "tuningConfig": {
+    "type": "kafka",
+    "reportParseExceptions": false
+  },
+  "ioConfig": {
+    "topic": "<env>.ml.observation.evidence.druid",
+    "consumerProperties": {
+      "bootstrap.servers": "broker_ip:broker_port"
+    },
+    "taskCount": "<ml_observation_evidence_taskCount>",
```

Comment:
topic, kafka broker ips and taskCount are incorrectly configured. Add proper variables for these fields.

Reply:
can you please share me the sample ingestion spec which is correct ? , because i go throughed the already existing ingestion spec of diksha.

Reply:
https://github.com/project-sunbird/sunbird-data-pipeline/tree/release-4.9.0/ansible/roles/druid-ingestion/templates - refer to ingestion spec in this folder and also move all your ingestion specs to this folder

Reply:
Resolved, Please check